package.json

```diff
@@ -22,7 +24,21 @@
   "scripts": {
     "test": "jest",
     "test:coverage": "jest --coverage",
-    "lint": "eslint . --cache --fix"
+    "lint": "eslint . --cache --fix",
+    "precommit": "lint-staged",
+    "prettier": "prettier --single-quote --trailing-comma es5 --write **/*.js *.js"
```

Comment:
we need to have the same prettier configuration (cli parameters) here and  at the lint-staged phase

Reply:
Fixed 60b800e